jsonschema-core.xml

```diff
@@ -455,6 +455,68 @@
 
         </section>
 
+        <section title="Principles">
+            <t>
+                JSON Schema is built on several principles, which both explain various design
+                choices, and provide guidance on how to extend the specification effectively.
+            </t>
+            <!-- Links will be added to relevant spec parts after the sections are reorganized
+                 to create the new schema loading section -->
+            <t>
+                <list>
+                    <t>
+                        All syntactically legal JSON documents with well-defined behavior are fully
+                        supported as instance data.  Notably, JSON objects with duplicate property
+                        names are syntactically legal but not well-defined.
+                    </t>
+                    <t>
+                        JSON Schema is defined over a data model, not JSON text.  Therefore
+                        JSON Schema can be used with any data format that can be reliably parsed
+                        into the data model.
```

Comment:
I'm guessing that this opens up the possibility for JSON Schema to be represented in YAML?

Reply:
It's always been a possibility.  The "data  model not txt" language is already present in the spec- it has its own section, even.   I think alternate representations are even explicitly called out (although maybe with CBOR rather than YAML, but I think a non-JSON case gets mentioned somewhere).  Although maybe only for instances and not schemas, but schemas are also instance so... 🤷‍♂

Reply:
Heck, I sometimes write my schemas in YAML and pre-build.

Reply:
I don't think there's anything to do here?  I'm going to mark this as resolved.